witness/src/Dscp/Snowdrop/Mode.hs

```diff
@@ -64,12 +67,14 @@ runSdMRead action = Lock.readingSDLock $ runSdM action
 -- | SdM runner, should be protected by some lock.
 runStateSdM
     :: (WitnessWorkMode ctx m, Lock.WithinReadSDLock)
-    => RememberForProof -> StateSdM a -> m a
-runStateSdM recForProof action = do
-    stateDBA <- SD.dmaAccessActions . flip nsStateDBActions recForProof <$> view (lensOf @SDVars)
-    unwrapSDBaseRethrow $
+    => StateSdM a -> m a
+runStateSdM action = do
+    stateDBA <- SD.dmaAccessActions . nsStateDBActions <$> view (lensOf @SDVars)
+    unwrapSDBaseRethrow $ do
+        initAVLStorage @AvlHash initAccounts
         SD.runERoCompIO @Exceptions stateDBA def action
 
 -- | Often used SdM runner that takes read lock.
-runStateSdMRead :: WitnessWorkMode ctx m => RememberForProof -> StateSdM a -> m a
-runStateSdMRead recForProof action = Lock.readingSDLock $ runStateSdM recForProof action
+runStateSdMRead :: WitnessWorkMode ctx m => StateSdM a -> m a
+runStateSdMRead action = Lock.readingSDLock $ runStateSdM action
+
```

Comment:
An empty line at the end of the file. How did CI allow you to do that? :thinking: